package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "brave-alert-lib",
-  "version": "1.0.0",
```

Comment:
[This comment is not directly related to your pull request.]

Hmmm, this is interesting to note. I didn't realize that it would show up here too. This version number change is coming from `package.json` in the `main` branch. We'll need to remember for the next release that we need to update it here too.

Reply:
It says 2 on my local repo in there, that's strange 🤔

Reply:
2 is good :+1:  That's what it should say. I just missed it when I was updating the version last time I merged into `main`.